NEWS.md

```diff
@@ -19,6 +19,7 @@
 ## misc
 
 - Add `configureMlr()` to `.onLoad()`, possibly fixing some edge cases (#2585) (@pat-s, #2637)
+- Allow ensemble filters to include multiple base filters of the same type (#2688)
```

Comment:
Please merge the latest master and put the item below `## filters - general`.

Reply:
OK.

Reply:
This should go below `## filters - general` not below `##misc`.

Reply:
That's weird. It is under filters - general in my version.